packages/lib/src/components/Pix/Pix.tsx

```diff
@@ -1,33 +1,34 @@
+import { h } from 'preact';
 import QRLoaderContainer from '../helpers/QRLoaderContainer';
 import CoreProvider from '../../core/Context/CoreProvider';
-import { h } from 'preact';
 import PixInput from './PixInput';
 import { cleanCPFCNPJ } from '../internal/SocialSecurityNumberBrazil/utils';
-import { PixProps } from './types';
+import { PixElementData, PixProps } from './types';
 
 class PixElement extends QRLoaderContainer<PixProps> {
     public static type = 'pix';
 
     public static defaultProps = {
+        showPayButton: false,
```

Comment:
Why was this changed? No other payment method defaults to `showPayButton: false`.

Reply:
You are right, I always thought Components had the same default has Dropin. I think we can keep it this way, no reason to change it now.